include/pinocchio/autodiff/casadi-algo.hpp

```diff
@@ -119,6 +121,16 @@ namespace pinocchio
         }
       }
 
+      long get_fun_operation_count()
+      {
+        return fun_operation_count;
+      }
+
+      long get_fun_derivs_operation_count()
```

Comment:
```suggestion
      casadi_int get_fun_derivs_operation_count() const
```

See get_fun_operation_count comment.

Reply:
Thank you Joris, this is now resolved.